repository/CargoPackageManager-Tests.package/CGOGitRepositoryCreatorTest.class/instance/testCanCreateRepositoryAndCloneItLater.st

```diff
@@ -1,23 +0,0 @@
-tests
-testCanCreateRepositoryAndCloneItLater
```

Comment:
why removing this test?

Reply:
Because this test is never going to work, it is using the old api.